PS.g4

```diff
@@ -184,8 +185,9 @@ fragment PI: '\\pi';
 fragment INFTY: '\\infty';
 SYMBOL: PI | INFTY;
 
-//PLACEHOLDER in one go
-PLACEHOLDER: '[!'[a-zA-Z][a-zA-Z0-9_]*'!]';
+fragment VARIABLE_CMD: '\\variable';
+fragment VARIABLE_SYMBOL: (GREEK_LETTER | LETTER) (GREEK_LETTER | LETTER | DIGIT)* (UNDERSCORE (GREEK_LETTER | LETTER | DIGIT)*)?;
```

Comment:
Would we want a `+` instead of a `*` after the subscript? Not sure if we want to allow an underscore without any characters following it.